.github/workflows/test-python.yml

```diff
@@ -7,18 +7,30 @@ jobs:
   run-unit-tests:
     
     name: Run all unit tests in codebase
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest]
+        python: [3.7]
+    runs-on: ${{ matrix.platform }}
     
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.7
+    - name: Set up Python ${{ matrix.python }}
       uses: actions/setup-python@v2
       with:
-        python-version: 3.7.1
+        python-version: ${{ matrix.python }}
+
+    # - if: matrix.platform == 'windows-latest'
```

Comment:
if we uncomment this, does it crash?

Reply:
No. I added this for windows support. We can put it back in -- we'd definitely need it to get windows working.